wordpress-mobile/WordPress-Android
# [Gutenberg] cancel upload on image block removal

This PR implements media upload cancellation for Gutenberg deleted blocks.

Gutenberg PR https://github.com/WordPress/gutenberg/pull/14391
Gutenberg mobile PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/736


To test:
1. start a draft
2. insert an image block
3. choose an image from the device so it starts uploading
4. once it starts uploading, tap on the block's trash icon
5. observe the foreground upload notification is also dismissed (meaning, the upload has been cancelled).

Test the same on Aztec (to delete an image, just place the cursor to the right of an image and hit backspace) and verify the behavior described in step 5 above.


Update release notes:
